bedrock/firefox/templates/firefox/nothing-personal/index.html

```diff
@@ -0,0 +1,149 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "firefox/base/base-protocol.html" %}
+
+{% from "firefox/nothing-personal/includes/browser-macro.html" import browser_border %}
+
+{% block page_css %}
+  {{ css_bundle('firefox-nothing-personal') }}
+{% endblock %}
+
+{% block page_title_prefix %}Nothing Personal{% endblock %}
+{% block page_title_suffix %}{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block content %}
+  <header class="c-page-header">
+    <div class="mzp-l-content c-page-header-inner">
+      <h1 class="c-page-header-logo-fx">
+        <img src="{{ static('img/firefox/nothing-personal/firefox-by-mozilla-wordmark.svg') }}" alt="" width="180" height="40" class="c-header-logo">
+        <span>Firefox by Mozilla</span>
+      </h1>
+
+      <div class="cta-group">
+        <p>Feeling browser-curious? We won’t tell. <span aria-hidden="true">>>>>></span></p>
+
+
+        <a href="{{ url('firefox.set-as-default.thanks')}}" class="is-not-default mzp-c-button mzp-t-lg" data-cta-type="button" data-cta-text="Set as default" data-cta-position="primary">Set as default</a>
+
+        <a href="https://blog.mozilla.org/en/firefox/" class="is-default mzp-c-button mzp-t-lg" data-cta-type="button" data-cta-text="Learn more" data-cta-position="primary">Learn more</a>
+
+        {{ download_firefox_thanks(alt_copy='Download Firefox', dom_id='protocol-nav-download-firefox', button_class='mzp-t-primary mzp-t-lg c-download-firefox-cta', download_location='primary') }}
+      </div>
+    </div>
+  </header>
+  <main class="c-page-main-content">
+    <section>
+    <div class="c-browser-windows">
+      <div class="c-layered-browser">
+        {% call browser_border(class='c-browser-window-top-wrapper mzp-l-content mzp-t-content-md') %}
+        <div class="c-browser-window-top">
+          <h2>We look forward to not getting to know you</h2>
+        </div>
+        {% endcall %}
+
+        {% call browser_border(class='c-browser-window-top-wrapper c-hidden-browser yellow mzp-l-content mzp-t-content-md') %}
+        <div class="c-browser-window-top">
+          <!-- Content is kept so it matches the size of the visible container above -->
+          <h2>We look forward to not getting to know you</h2>
+        </div>
```

Comment:
**suggestion:** is it worth adding `aria-hidden` to these duplicated windows to avoid  the content being repeated to screen readers several times?

Alternatively you could probably re-work this using CSS and absolute positioning to avoid needing to repeat the content. But given the time we have, perhaps this is easier.

Reply:
Since the content mentioned here has `visibility: hidden` on its CSS, then screenreaders will avoid these windows :) 

> These styles will hide content from all users. The content is removed from the visual flow of the page and is ignored by screen readers. Do not use this CSS if you want the content to be read by a screen reader. But DO use it for content you want hidden from all users.

https://webaim.org/techniques/css/invisiblecontent/

Reply:
Ah, I missed they had visibility:hidden 👍